gm4_enchantment_extractors/data/enchantment_extractors/functions/destroy.mcfunction

```diff
@@ -0,0 +1,3 @@
+summon item ~ ~1 ~ {Item:{id:enchanting_table,Count:1},Passengers:[{id:item,Item:{id:"hopper",Count:1}},{id:item,Item:{id:"book",Count:7}},{id:item,Item:{id:"writable_book",Count:1}}]}
+data remove entity @e[type=item,distance=..3,limit=1,nbt={Item:{tag:{display:{Name:"{\"text\":\"Enchantment Extractor\"}"}}}}] Item.tag.display
```

Comment:
I'm kinda outta the loop here, what's this line for?

Reply:
It removes the display tag so it will stack with other hoppers